mlflow/tracking/fluent.py

```diff
@@ -1072,6 +1072,59 @@ def log_table(
     MlflowClient().log_table(run_id, data, artifact_file)
 
 
+@experimental
+def load_table(
+    artifact_file: str,
+    run_ids: Optional[List[str]] = None,
+    extra_columns: Optional[List[str]] = None,
+) -> "pandas.DataFrame":
+    """
+    Load a table from MLflow Tracking as a pandas.DataFrame. The table is loaded from the
+    specified artifact_file in the specified run_ids. The extra_columns are columns that
+    are not in the table but are augmented with run information and added to the DataFrame.
+
+    :param artifact_file: The run-relative artifact file path in posixpath format to which
+                          table to load (e.g. "dir/file.json").
+    :param run_ids: Optional list of run_ids to load the table from. If no run_ids are specified,
+                    the table is loaded from all runs in the current experiment.
+    :param extra_columns: Optional list of extra columns to add to the returned DataFrame
+                          with extra as a prefix. For example, if extra_columns=["run_id"], then
+                          the returned DataFrame will have a column named extra_run_id.
+
+    :return: pandas.DataFrame
```

Comment:
Can we add more detail here and indicate that this throws if the artifact file is not found in any runs?